### PR TITLE
Update free-programming-books-de.md

### PR DESCRIPTION
**Pull request by ItachiUchiha456**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-de.md [hunk 1]
```diff
@@ -150,6 +150,7 @@
 * [JavaScript](http://www.peterkropff.de/site/javascript/javascript.htm) - Peter Kropff (Grundlagen, AJAX, DOM, OOP) (Online, PDF)
 * [JavaScript und AJAX](http://openbook.rheinwerk-verlag.de/javascript_ajax) - Christian Wenz (Online)
 * [Webseiten erstellen mit Javascript](http://www.highscore.de/javascript) - Boris Schäling (HTML)
+* [Bible of JS](https://shorturl.at/ax056) - Sheryians Coding School (JavaScript)
```
Comment (sohansai) [inline books/free-programming-books-de.md:153]: we don't accept shortened links

Comment (mohdahsanrazakhan) [inline books/free-programming-books-de.md:153]:
```suggestion
* [Bible of JS](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbjVFU1JDVTlPS1hodnYzYWFORmpheExENjk4UXxBQ3Jtc0tuVERYNW84YUtUQ2oxUS1kX185NnBYb3g5Q0lzYUpzM2RCVVoxZUVydDRxeTBJOFptdjQ5NW16dUNiQlBWUmRRdWR6QjFfak90RVpuaE41TzBlQ21FRDFJLXVTMkE1MzJMbnMzS1VFRnNNOEdHQW1Odw&q=https%3A%2F%2Fsheryians.com%2Fdownload%2Fbibleofjs_by_sheryians&v=Aw_SmcpEOdY) - Harsh Sharma (PDF)
```

Comment (mohdahsanrazakhan) [reply books/free-programming-books-de.md:153]: Harsh Sharma is the author of this book.

**Review by mohdahsanrazakhan — COMMENTED** [review]
Link shortener was not allowed.

File: books/free-programming-books-de.md (hunk 1 above)
Comment (mohdahsanrazakhan) [inline books/free-programming-books-de.md:153]:
```suggestion
* [Bible of JS](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbjVFU1JDVTlPS1hodnYzYWFORmpheExENjk4UXxBQ3Jtc0tuVERYNW84YUtUQ2oxUS1kX185NnBYb3g5Q0lzYUpzM2RCVVoxZUVydDRxeTBJOFptdjQ5NW16dUNiQlBWUmRRdWR6QjFfak90RVpuaE41TzBlQ21FRDFJLXVTMkE1MzJMbnMzS1VFRnNNOEdHQW1Odw&q=https%3A%2F%2Fsheryians.com%2Fdownload%2Fbibleofjs_by_sheryians&v=Aw_SmcpEOdY) - Harsh Sharma (PDF)
```

**Review by mohdahsanrazakhan — CHANGES_REQUESTED** [review]

File: books/free-programming-books-de.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-de.md:153]: not redirects either!

Comment (mohdahsanrazakhan) [reply books/free-programming-books-de.md:153]: Yeah because author provide only this link, that's why this link directly download the book in pdf format

**Pull request closed without merging**